includes/rest/class-outbox.php

```diff
@@ -105,7 +105,8 @@ public static function user_outbox_get( $request ) {
 			);
 
 			foreach ( $posts as $post ) {
-				$post = Post::transform( $post )->to_object();
+				$transformer = \Activitypub\Transformer\Transformers_Manager::instance()->get_transformer( $post );
```

Comment:
```suggestion
				$transformer = Transformers_Manager::instance()->get_transformer( $post );
```

you do not have to use the full path (with namespaces) when using `use`.